src/qtwebengine/viewwindow.cpp

```diff
@@ -115,9 +108,7 @@ QMenu * ViewWindow::getContextMenu( const QUrl & link, QWidget * parent )
         {
             m_contextMenuLink = createStandardContextMenu( parent );
             m_contextMenuLink->addSeparator();
-
```

Comment:
Don't do this.

Reply:
Some files are indented with spaces, others with tabs, and a few have both spaces and tabs. There are also tail spaces. It's hard not to screw something up.

Reply:
This is why proofreading the .diff file is recommended. If this creates an issue, the proper solution is to submit a bug report, so this is fixed in the source tree.